tests/onnx/test_nncf_graph_builder.py

```diff
@@ -14,53 +14,109 @@
 import os
 import pytest
 
+import torch
+from torchvision import models
+import onnx
+
 from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
+from nncf.experimental.onnx.model_normalizer import ONNXModelNormalizer
 
-from tests.onnx.models import LinearModel
-from tests.onnx.models import MultiInputOutputModel
-from tests.onnx.models import ModelWithIntEdges
+from tests.onnx.models import ALL_MODELS
+from tests.common.helpers import TEST_ROOT
+from tests.onnx.quantization.common import ModelToTest
+from tests.onnx.quantization.common import check_nx_graph
 
 import networkx as nx
 
-TEST_MODELS = [LinearModel, MultiInputOutputModel, ModelWithIntEdges]
+TEST_MODELS = ALL_MODELS
```

Comment:
Should have just used the original name and refactored the usages of `TEST_MODELS` instead.

Reply:
Done